src/azure-cli/azure/cli/command_modules/vm/generated/_help.py

```diff
@@ -52,3 +52,85 @@
     type: command
     short-summary: "Delete an SSH public key."
 """
+
+helps['sig'] = """
+    type: group
+    short-summary: Manage gallery with vm
+"""
+
+helps['sig image-definition'] = """
+    type: group
+    short-summary: Manage gallery with vm sub group image-definition
+"""
+
+helps['sig image-definition list-shared'] = """
+    type: command
+    short-summary: "List shared gallery images by subscription id or tenant id."
```

Comment:
Why does `short-summary` sometimes use double quotation marks?

Reply:
this is auto-generated part.